source/extensions/network/dns_resolver/apple/apple_dns_impl.cc

```diff
@@ -335,6 +336,7 @@ void AppleDnsResolverImpl::PendingResolution::onDNSServiceGetAddrInfoReply(
     parent_.chargeGetAddrInfoErrorStats(error_code);
 
     pending_response_.status_ = ResolutionStatus::Failure;
+    pending_response_.details = absl::StrCat(error_code);
```

Comment:
maybe we should append `onDNSServiceGetAddrInfoReply` or even just `on_reply` to the details here to know it happened in the reply callback ?

Reply:
WDYT?